docs/index.md

```diff
@@ -3,28 +3,34 @@ myst:
   html_meta:
     "description": "The Volto user interface (UI) is a React-based frontend for the Plone CMS. It is the default user interface starting with the release of Plone 6."
     "property=og:description": "The Volto user interface (UI) is a React-based frontend for the Plone CMS. It is the default user interface starting with the release of Plone 6."
-    "property=og:title": "Volto UI"
-    "keywords": "Volto, Plone, frontend, user interface, React"
+    "property=og:title": "Seven"
+    "keywords": "Seven, Plone, frontend, user interface, React"
 ---
 
 (volto-index-label)=
 
-# Volto UI
+# Seven
 
-The Volto user interface (UI) is a React-based frontend for the [Plone content management system](https://plone.org).
-It is the default user interface starting with the release of Plone 6.
+The Seven user interface (UI) is a React-based frontend for the [Plone content management system](https://plone.org).
 
-Volto provides an attractive proposition: integration with the modern frontend development world, access to the huge ecosystem of React libraries and add-ons, combined with the ability to use the mature Plone CMS backend as a development platform.
+Seven is the codename for the successor of Volto.
+The codename is _temporary_, the final name of the product has to be decided in the future, by the community when the release happens.
+It is being developed [in its own branch](https://github.com/plone/volto/tree/7) along side Volto (in main branch) as it happens during the next months.
```

Comment:
I'm still not clear what branches are under active development. I think `17.x.x` is in maintenance support, not development, so maybe drop that one? I'm also not clear about `18.x.x` because it doesn't exist yet, and I assume that v19 will develop on `main`.

```suggestion
It is under active development [in its own branch `7`](https://github.com/plone/volto/tree/7) in parallel with Volto in the `main` branch and versioned branches `17.x.x` and `18.x.x`.
```

Reply:
Now `18.x.x` does exist. I will update the Seven branch name, since it has been already renamed. There's no point in mentioning `17.x.x`.